frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -1,22 +1,25 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import * as _ from 'lodash-es';
+import { DragDropContext } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
 
-import { ValueFromPair } from './value-from-pair';
+import { PairElement } from './pair-element';
```

Comment:
Vendor bundle increases by 200kB by adding this library (look at the "Performance" PR check). What we do in this case is use `AsyncComponent` to load the `PairElement` component, which will bundle its heavy dependencies into an async chunk using dynamic `import()`. [See `EditYAMLPage` for an example](https://github.com/openshift/console/blob/612f933e2cab16af2fdeecdb5cd33df0a62eb9b0/frontend/public/components/create-yaml.tsx#L53).

Reply:
Thanks Alec. Will take a look at AsyncComponent.

Reply:
@alecmerdler Updated name-value-editor to use the AsyncComponent to load the PairElement Component instead of importing it directly.